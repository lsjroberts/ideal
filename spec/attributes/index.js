import { color, el, font, rgb, text } from 'ideal-ui';

export default el([font.bold(), color(rgb(1, 0, 0))], text`hello`);
