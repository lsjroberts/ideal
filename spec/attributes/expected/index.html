<!DOCTYPE html>
<html>
  <head>
    <title>text</title>
    <style type="text/css">
      html,
      body {
        margin: 0;
        padding: 0;
        height: 100%;
      }

      .s {
        position: relative;
        border: none;
        flex-shrink: 0;
        display: flex;
        flex-direction: row;
        flex-basis: auto;
        resize: none;
        box-sizing: border-box;
        margin: 0;
        padding: 0;
        border-width: 0;
        border-style: solid;
        font-size: inherit;
        color: inherit;
        font-family: inherit;
        line-height: 1;
        font-weight: inherit;
        text-decoration: none;
        font-style: inherit;
      }

      .s.e {
        display: flex;
        flex-direction: column;
        white-space: pre;
      }
    </style>
    <style type="text/css">
      .w7 {
        font-weight: 700;
      }

      .fc-255-0-0 {
        color: rgb(255, 0, 0);
      }
    </style>
  </head>
  <body>
    <div class="s e w7 fc-255-0-0">hello</div>
  </body>
</html>
